### DISCUSSION TITLE
Gene not in set and cannot be visualized

### DISCUSSION THREAD

**Discussion by ccomo0730** [stuart-lab/signac#1512] [General]
Hello! I am trying to show a link between the gene Sox2 and the upstream gene Sox2ot. Sox2ot is a long non coding RNA, but when I try to visualize it, it isn't found. Is there any way to find this gene or add it?
chr3-34614864-34732142

Thanks!

**Comment by timoast** [comment]
https://github.com/stuart-lab/signac/issues/1513
